app/ml_models/controllers.py

```diff
@@ -61,10 +61,10 @@ def prepare_data(as_generator=False, text_encoder=False):
     # filenames.append('10-19-20s_706posts-400-1N1S1L2U2L(2017-01-11 20:56:06.286)')
     # filenames.append('10-19-30s_705posts-400-1N1S1L2U2L(2017-01-11 20:56:06.286)')
     # filenames.append('10-19-40s_686posts-400-1N1S1L2U2L(2017-01-11 20:56:06.286)')
-    # filenames.append('10-19-adults_706posts-400-1N1S1L2U2L(2017-01-11 20:56:06.286)')
-    # filenames.append('editorial-brown-400-1N1S1L2U2L(2017-01-17 17:30:39.071)')
+    filenames.append('10-19-adults_706posts-400-1N1S1L2U2L(2017-01-11 20:56:06.286)')
```

Comment:
If you need more diverse datasets from NLTK, we can create them with the speedy-affect-scorer project.